spacy_llm/pipeline/llm.py

```diff
@@ -21,6 +21,20 @@
 logger = logging.getLogger("spacy_llm")
 logger.addHandler(logging.NullHandler())
 
+DEFAULT_MODEL_CONFIG = {
+    "@llm_models": "spacy.GPT-3-5.v1",
```

Comment:
Should be v2 when https://github.com/explosion/spacy-llm/pull/236/files is merged